pkg/vacuum/vacuum.go

```diff
@@ -60,40 +60,48 @@ import (
 )
 
 var (
-	tablesVacuumedTotal = prometheus.NewCounter(prometheus.CounterOpts{
+	numberVacuumConnections = prometheus.NewGauge(prometheus.GaugeOpts{
 		Namespace: util.PromNamespace,
 		Subsystem: "vacuum",
-		Name:      "tables_vacuumed_total",
-		Help:      "Total number of tables vacuumed by the Promscale vacuum engine.",
+		Name:      "number_vacuum_connections",
```

Comment:
We are repeating `vacuum` used in subsystem.
```suggestion
		Name:      "num_connections",
```